docs/easybuild-v5/overview-of-changes.md

```diff
@@ -48,7 +46,12 @@ feedback then please comment in the GitHub issue for the proposal.
 
 Various significant enhancements are included in EasyBuild v5.0, including:
 
-* *(no significant enhancements implemented so far in the `5.0.x` branches)*
+* [enable `--trace` by default][eb5_trace]
```

Comment:
I guess you could debate whether this is an enhancement or not, rather than just a changed default, but fine as is.